pymove/preprocessing/segmentation.py

```diff
@@ -38,45 +41,88 @@ def bbox_split(bbox, number_grids):
     const_lon = abs(abs(lon_max) - abs(lon_min)) / number_grids
     print("const_lat: {}\nconst_lon: {}".format(const_lat, const_lon))
 
-    move_data = pd.DataFrame(columns=["lat_min", "lon_min", "lat_max", "lon_max"])
+    move_data = pd.DataFrame(
+        columns=["lat_min", "lon_min", "lat_max", "lon_max"]
+    )
     for i in range(number_grids):
-        move_data = move_data.append({"lat_min": lat_min,
-                                      "lon_min": lon_min + (const_lon * i),
-                                      "lat_max": lat_max,
-                                      "lon_max": lon_min + (const_lon * (i + 1))},
-                                     ignore_index=True)
+        move_data = move_data.append(
+            {
+                "lat_min": lat_min,
+                "lon_min": lon_min + (const_lon * i),
+                "lat_max": lat_max,
+                "lon_max": lon_min + (const_lon * (i + 1)),
+            },
+            ignore_index=True,
+        )
 
     return move_data
 
 
+def _drop_single_point(move_data, label_new_tid, label_id):
+    """
+    Removes trajectory with single point
+
+    Parameters
+    ----------
+    move_data: dataframe
+        dataframe with trajectories
+    label_new_tid : String
+        The label of the column containing the ids of the formed segments. Is the new splitted id.
+    label_id : String
+         Indicates the label of the id column in the user"s dataframe.
+    """
+    shape_before_drop = move_data.shape
+    idx = move_data[move_data[label_new_tid] == -1].index
+    if idx.shape[0] > 0:
+        print("...Drop Trajectory with a unique GPS point\n")
+        ids_before_drop = move_data[label_id].unique().shape[0]
+        move_data.drop(index=idx, inplace=True)
+        print(
+            "...Object - before drop: {} - after drop: {}".format(
+                ids_before_drop, move_data[label_id].unique().shape[0]
+            )
+        )
+        print(
+            "...Shape - before drop: {} - after drop: {}".format(
+                shape_before_drop, move_data.shape
+            )
+        )
+    else:
+        print("...No trajs with only one point.", move_data.shape)
+
+
 def by_dist_time_speed(
-        move_data,
-        label_id=TRAJ_ID,
-        max_dist_between_adj_points=3000,
-        max_time_between_adj_points=7200,
-        max_speed_between_adj_points=50.0,
-        drop_single_points=True,
-        label_new_tid="tid_part",
-        inplace=True
+    move_data,
+    label_id=TRAJ_ID,
+    max_dist_between_adj_points=3000,
+    max_time_between_adj_points=7200,
+    max_speed_between_adj_points=50.0,
+    drop_single_points=True,
+    label_new_tid=TID_PART,
+    inplace=True,
 ):
-    """Splits the trajectories into segments based on distance, time and speed.
+    """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Section underline should match the length of its name (Expected 7 dashes in section 'Returns', got 6) (D409)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5452500)